bundles/core/src/test/resources/image/test-content.json

```diff
@@ -284,6 +284,36 @@
                             "jcr:uuid"          : "22a0a463-9614-48c5-8215-a35ffc7d1089"
                         }
                     }
+                },
+                "image22": {
+                    "jcr:primaryType"    : "nt:unstructured",
+                    "jcr:createdBy"      : "admin",
+                    "cq:lastModified"    : "2017-05-15T17:56:17.756+01:00",
+                    "sling:resourceType" : "core/wcm/components/image/v1/image",
```

Comment:
Are we also testing image/v2?

Reply:
We're not actually testing the image component's implementation, but the AIS' behaviour.